### PR TITLE
Taxi/experimental/map

### PR DESCRIPTION
**Pull request by kikokeko88**

### CODE REVIEW

File: client_taxi_driver/androidApp/src/androidMain/assets/bing_map/map/script.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
+var map,directionsManager;
+function getMap() {
+map = new Microsoft.Maps.Map('#myMap', {
+    credentials: 'Access_token',
+    center: new Microsoft.Maps.Location(0,0),
```
Comment (AzizaHelmy) [inline client_taxi_driver/androidApp/src/androidMain/assets/bing_map/map/script.js:5]: why this line?
i removed it from delivery map as it casues a new point on the center of the map

File: client_taxi_driver/androidApp/src/androidMain/assets/bing_map/map/script.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    mapTypeId: Microsoft.Maps.MapTypeId.road,
+    minZoom: 4,
+    maxZoom: 18
+});
+    var center = map.getCenter();
```
Comment (AzizaHelmy) [inline client_taxi_driver/androidApp/src/androidMain/assets/bing_map/map/script.js:10]: why this line?

**Review by Youssef-Ramadan-Y — APPROVED** [review]

**Review by Asiasama710 — APPROVED** [review]

**Review by ahmednasserzaza — APPROVED** [review]

File: client_taxi_driver/shared/build.gradle.kts [hunk 3]
```diff
@@ -61,6 +62,7 @@ kotlin {
                 implementation(libs.kotlinx.datetime)
 
                 api(libs.moko.geo)
+                api("com.mohamedrejeb.calf:calf-ui:0.2.0")
```
Comment (AzizaHelmy) [inline client_taxi_driver/shared/build.gradle.kts:65]: why add this dependency again?

**Review by devsadeq — APPROVED** [review]

File: client_taxi_driver/shared/src/commonMain/kotlin/presentation/base/map.kt [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.ui.Modifier
+import com.mohamedrejeb.calf.ui.web.WebView
+import com.mohamedrejeb.calf.ui.web.rememberWebViewState
+
+@Composable
+fun Map() {
```
Comment (AzizaHelmy) [inline client_taxi_driver/shared/src/commonMain/kotlin/presentation/base/map.kt:11]: why this file?

**Review by AzizaHelmy — APPROVED** [review]

**Pull request merged**